prow/config/config.go

```diff
@@ -108,6 +108,10 @@ type ProwConfig struct {
 
 	// Pub/Sub Subscriptions that we want to listen to
 	PubSubSubscriptions PubsubSubscriptions `json:"pubsub_subscriptions,omitempty"`
+
+	// AllowCancellations enables aborting presubmit jobs for commits that
+	// have been superseded by newer commits in Github pull requests.
+	AllowCancellations bool `json:"allow_cancellations,omitempty"`
```

Comment:
Can we DRY this out by embedding a type?

Reply:
Do you mean to add a config type for knative build controller?

Reply:
Yes

Reply:
Yes

Reply:
I moved it in a build controller config type.